### PR TITLE
New Jukebox music

### PR DESCRIPTION
**Pull request by Katarn1933**
## About the PR
Has add music track (for now as experiment how to do it) to jukebox. Music is licensed under CC-BY-3.0 and should be fine to use for out purposes

## Why / Balance
So we can have bit more variety of music in jukebox

## How to test
play it under "Silverman Sound Studios - Later Alligator" on jukebox

## Media
New soundtrack for jukebox
https://soundcloud.com/silvermansound/later-alligator

[x ] I have added screenshots/videos to this PR showcasing its changes ingame, or this PR does not require an ingame showcase
## Breaking changes
N\A
## Changelog
New swampy tune for jukebox
:cl: Katarn1933

- add: New swampy tune for jukebox

### CODE REVIEW

**Comment by dvir001** [comment]
Use _NF folders

File: Resources/Audio/Jukebox/attributions.yml [hunk 1]
```diff
@@ -24,4 +24,9 @@
 - files: ["sunset.ogg"]
   license: "CC-BY-SA-3.0"
   copyright: "Sunset by PigeonBeans. Exported in Mono OGG."
-  source: "https://soundcloud.com/pigeonbeans/sunset"
\ No newline at end of file
+  source: "https://soundcloud.com/pigeonbeans/sunset"
+
+- files: ["lateraligator.ogg"]
+  license: "CC-BY-3.0"
+  opyright: "Later Alligator By Silverman Sound Studios. Converted to mono ogg"
```
Comment (whatston3) [inline Resources/Audio/Jukebox/attributions.yml:31]:
```suggestion
  copyright: "Later Alligator By Silverman Sound Studios. Converted to mono ogg"
```

**Comment by Katarn1933** [comment]
> Use _NF folders 

I can't quite find existing songs in _NF folder. should I create Jukebox folder under /Audio/_NF/ and store it there?

**Comment by dvir001** [comment]
Wait, still not under _NF

**Pull request merged**